editor/static/scripts/settings.js

```diff
@@ -1,6 +1,6 @@
 import {saveState} from "./state_handler";
 
```

Comment:
These settings somehow need to be persistent. Maybe save a JSON copy under ~/.scheme-editor/config or something.